src/main/java/se/sundsvall/webmessagecollector/integration/opene/OpenEClient.java

```diff
@@ -1,9 +1,11 @@
 package se.sundsvall.webmessagecollector.integration.opene;
 
+import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestParam;
 
+@CircuitBreaker(name = "open-e")
```

Comment:
Can we reuse the registration-ID perhaps (haven't looked, but probably defined in Configuration-class?)

Reply:
No enum present.